src/net/rpc.rs

```diff
@@ -110,7 +128,7 @@ impl fmt::Debug for ConversationHttp {
 impl PeerInfoData {
     pub fn from_db(burnchain: &Burnchain, burndb: &mut BurnDB, peerdb: &mut PeerDB) -> Result<PeerInfoData, net_error> {
         let burnchain_tip = BurnDB::get_canonical_burn_chain_tip(burndb.conn()).map_err(net_error::DBError)?;
-        let local_peer = PeerDB::get_local_peer(peerdb.conn()).map_err(net_error::DBError)?;
+        let local_peer = PeerDB::get_local_peer(peerdb.conn()).map_err(net_error::DBError).unwrap();
```

Comment:
Please revert this and preserve the `?`.

Reply:
Oops, left this in accidentally— was trying to figure out the source of a thrown error.

Reply:
Reverted in 5951759a789563a6ecb1ae3800fc0706944b02ec